guide/onboarding/introduction.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: guide
+title: Onboarding
+description: An overview of some of the concepts to consider when building onboarding experiences.
+nav_order: 2
+has_children: true
+permalink: /guide/onboarding/introduction/
+---
+
+<!--
+
+Editor's notes
+
+A brief introduction and summary of all pages in this section. The idea is that readers
+scan this page to get an overview of the section and then decide which topics to dive into.
+
+-->
+
+# Onboarding
+
+Effective onboarding can make or break someone’s first bitcoin experience. Therefore, it is important to identify your target audience when building wallet-products and design specifically with that audience in mind. 
+
+Some new users bring years of traditional financial experience to bitcoin. They have bank accounts, debit cards, and ingrained ideas about what money is. This makes them tricky to onboard. Users might assume that the patterns and habits from prior experience will apply, so convincing them to use a different financial system might be better in the product. 
+
+On the other hand, more than four billion people have no exposure to traditional financial institutions. This, along with designing for younger people whose ideas about money aren’t fully formed, represents a very different challenge.
```

Comment:
Backing this up with research would be great. No worries if there is no good reference.

Reply:
I would strike or rethink this phrase: "This makes them tricky to onboard."
Them, is basically every new user to bitcoin and this reads as them being unusual.

Reply:
The number seems to be closed to 2 Billion as well according to the World Bank - https://globalfindex.worldbank.org/sites/globalfindex/files/chapters/2017%20Findex%20full%20report_chapter2.pdf

Although I feel like I have definitely seen larger numbers. 

@danielnordh think I agree, will update 🚧

Reply:
Updated, let me know your thoughts